src/app/(homepage)/HomepageTabs.js

```diff
@@ -96,7 +99,12 @@ function ContentPanel({ type }) {
         <Typography mb={3} variant="body1" color="text.secondary">
           {data.description}
         </Typography>
-        <Button variant="outlined" color="orange" size="large">
+        <Button
```

Comment:
You bind a specific behaviour to a generic component.
That makes this component not reusable.